Fix ballerina test debug mode does not print debug suspended msg

rasika opened the pull request:
## Purpose
> Fix tests in debug mode doesn't give any output until a remote debug client is connected 

Fixes #20429

## Related PRs
https://github.com/ballerina-platform/ballerina-lang/pull/20432

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [x] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

MaryamZi commented:
Can we add back the prompt to start a remote client too https://github.com/ballerina-platform/ballerina-lang/commit/f51121c262589623ebdf4df063f16933fbc54e9a#diff-6de39288fa987618d16a0a32fb67ea1bL168?  

Relevant issue: https://github.com/ballerina-platform/ballerina-lang/issues/20419

NipunaRanasinghe commented on distribution/zip/jballerina/bin/ballerina:
```diff
@@ -159,13 +159,16 @@ done
+      if [ -n "$JAVA_OPTS" ]; then
+        echo "Warning !!!. User specified JAVA_OPTS may interfere with BAL_JAVA_DEBUG"
+      fi
+      if [ -n "${DEBUG_OPTS+set}" ]; then
```
How about being more specific by using something like `BAL_DEBUG_OPTS` as the env var name instead, just to avoid confusions when working in environments which have multiple debugger related entities? :)

nadeeshaan replied:
+1 for the suggestion

rasika replied:
+1 that makes sense to me. Will update the PRs

rasika commented:
> Can we add back the prompt to start a remote client too [f51121c#diff-6de39288fa987618d16a0a32fb67ea1bL168](https://github.com/ballerina-platform/ballerina-lang/commit/f51121c262589623ebdf4df063f16933fbc54e9a#diff-6de39288fa987618d16a0a32fb67ea1bL168)?
> 
> Relevant issue: #20419

Removed it since it is redundant. The issue here is we have added `quiet=y` into JVM debug options. I have removed it from the default case. Thus you will get a console print as below;

<img width="595" alt="Screenshot 2020-01-07 at 10 36 23" src="https://user-images.githubusercontent.com/1448489/71870136-ce772380-313a-11ea-8058-4ca09fac6b57.png">
